### ISSUE TITLE
Can the author explain what the "get_field_values" function does?

### ISSUE THREAD

**Issue by wangyuanbiubiubiu** [Anttwo/SuGaR#65]
`    def get_field_values(self, x, gaussian_idx=None, 
                    closest_gaussians_idx=None,
                    gaussian_strengths=None, 
                    gaussian_centers=None, 
                    gaussian_inv_scaled_rotation=None,
                    return_sdf=True, density_threshold=1., density_factor=1.,
                    return_sdf_grad=False, sdf_grad_max_value=10.,
                    opacity_min_clamp=1e-16,
                    return_closest_gaussian_opacities=False,
                    return_beta=False,):`
I can't understand this function very well, can the author provide some help？

**Comment by wangyuanbiubiubiu** [comment]
In fact, I have been puzzled by a sentence in the article, why the density function is determined by the Gaussian value, rather than the weight value of T*a

**Comment by wangyuanbiubiubiu** [comment]
`sdf_estimation_loss = (sdf_values - sdf_estimation.abs()).abs() / sdf_sample_std`
Here, both sdf_values and sdf_estimation actually return the gradients, is that what's expected? One should be an measurement value
